pkg/vm/vm.go

```diff
@@ -80,19 +80,13 @@ type VM struct {
 	keys map[string]*keys.PublicKey
 }
 
-// InteropFuncPrice represents an interop function with a price.
-type InteropFuncPrice struct {
-	Func  InteropFunc
-	Price int
-}
-
 // New returns a new VM object ready to load .avm bytecode scripts.
 func New() *VM {
 	vm := &VM{
-		interop:   make(map[string]InteropFuncPrice),
-		getScript: nil,
-		state:     haltState,
-		istack:    NewStack("invocation"),
+		getInterop: make([]InteropGetterFunc, 0, 3), // 3 functions is typical for our default usage.
```

Comment:
hm and what are these 3 functions?

Reply:
`getDefaultVMInterop`, `getSystemInterop`, `getNeoInterop`. I'm actually not sure it's worth having this as a slice, but it allows to make `VM` behaviour closer to what it had been before the change.

Reply:
I see, not that obvious about these interops, probably we can mention that in godoc/comment?

Reply:
I've updated the commit message.